src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,17 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
```

Comment:
same as before
You should fix all the same issues accross the project when one of them was highlighted

Read more about fragment usage: https://react.dev/reference/react/Fragment